# Clase 14

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

$$ $$

### M. R. U. A.  

En el MRUA, por el momento, las gráficas que nos importarán son:
* v vs. t 
* a vs. t.

**1) ¿Qué	significa	MRUA?	Escribe	sus	formulas.**

Significa que **ahora** la aceleración es constante.

**¿Qué es aceleración?** La rapidez de la rapidez.

"Cómo cambia la velocidad en el tiempo."

$$ a=\dfrac{v_f-v_i}{t} $$

Las fórmula importantes:

$t$ : tiempo transcurrido

$d$ : distancia

$v_i$ : rapidez inicial

$v_f$ : rapidez final

$a$ : aceleración

$$ a=9.81\ \dfrac{m}{s^2}=\dfrac{9.81\ \frac{m}{s}}{s} $$

La aceleración debida a la gravedad me dice que, si yo caigo, la rapidez con la que caigo aumenta $9.81 \dfrac{m}{s}$ cada segundo.

**Extra1) Un automóvil  mantiene una aceleración constante de $8\ \dfrac{m}{s^2}$. Si su rapidez inicial era de $20\ \dfrac{m}{s}$, ¿cuál será su rapidez después de $6 s$?**

* Datos:
    * $t=6s$
    * $v_i=20m/s$
    * $v_f=?$
    * $a=8m/s^2$

* Fórmula:

$$ a=\dfrac{v_f-v_i}{t} $$

$$ at=v_f-v_i $$

$$ v_f=at+v_i $$

* Sustitución:

$$ v_f=(8)(6)+20=68\ \dfrac{m}{s} $$

**3)La	siguiente	gráfica	representa	un	MRUA	calcula	la	aceleración	y	la	distancia	recorrida	a	los	6	s:**

* Datos:
    * $t=6s$
    * $d=300m$
    * $v_i=20m/s$
    * $v_f=80m/s$
    * $a=?$

* Fórmulas:
    1. $d=\left(\dfrac{v_i+v_f}{2}\right)t$

    2. $v_f=v_i+a\cdot t$

    3. $d=v_i\cdot t+\dfrac{1}{2}\cdot a\cdot t^2$

    4. $d=v_f\cdot t-\dfrac{1}{2}\cdot a\cdot t^2$

    5. ${v_f}^2={v_i}^2+2\cdot a\cdot d$

* Sustitución

$$d=\left(\dfrac{20m/s+80m/s}{2}\right)(6s)=300m$$

$$v_f=v_i+a\cdot t$$


$$ v_f-v_i=at $$

$$ a=\dfrac{v_f-v_i}{t} $$

$$ a=\dfrac{80m/s-20m/s}{6s}=10m/s^2 $$

**4)Una	moto	con	una	velocidad	inicial	de	6	m/s	se	acelera	uniformemente	a	razón	de	1.5	m/s2	durante	8	
segundos.	a)	Calcula	la	distancia	que	recorre	la	moto	durante	ese	tiempo.	b)	¿Qué	velocidad	lleva	la	
moto	a	los	8	segundos?**

* Datos:
    * $t=8s$
    * $d=96m$
    * $v_i=6m/s$
    * $v_f=?$
    * $a=1.5m/s^2$

* Fórmulas:
    1. $d=\left(\dfrac{v_i+v_f}{2}\right)t$

    2. $v_f=v_i+a\cdot t$

    3. $d=v_i\cdot t+\dfrac{1}{2}\cdot a\cdot t^2$

    4. $d=v_f\cdot t-\dfrac{1}{2}\cdot a\cdot t^2$

    5. ${v_f}^2={v_i}^2+2\cdot a\cdot d$

* Sustitución:

$$d=(6m/s)\cdot(8s)+\dfrac{1}{2}\cdot (1.5m/s^2)\cdot (8s)^2=48m+48m=96m$$

$${v_f}^2={v_i}^2+2\cdot a\cdot d$$

$${v_f}=\sqrt{{v_i}^2+2\cdot a\cdot d}$$

$${v_f}=\sqrt{({6m/s})^2+2\cdot (1.5m/s^2)\cdot (96m)}$$

$${v_f}=\sqrt{324m^2/s^2}=18m/s$$

$$v_f=v_i+a\cdot t$$

$$v_f=(6m/s)+(1.5m/s^2)\cdot(8s)=18m/s$$

**5) Si	un	avión	que	aterriza	a	216	km/h	se	detiene	después	de	recorrer	1200	m.	Calcula	a)	la	aceleración	
del	avión,	b)	el	tiempo	que	tarda	en	detenerse.**

### Caída libre y tiro vertical

**1) ¿Cómo	sabes	si	un	cuerpo	está	en	caída	libre?	Escribe	sus	formulas	considerando	lo	que	acabas	de	escribir.**


Si la aceleración de un cuerpo coincide con la acelaración debida a la gravedad, éste está en caída libre.

* Caída libre es un tipo de MRUA
* Esto significa que tendremos las mismas fórmulas de MRUA pero con
    
    $$a\to -g\qquad d\to h$$

* Fórmulas:
    1. $h=\left(\dfrac{v_i+v_f}{2}\right)t$

    2. $v_f=v_i-g\cdot t$

    3. $h=v_i\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h=v_f\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${v_f}^2={v_i}^2-2\cdot g\cdot h$

* Caída libre, $v_i=0m/s$
* Tiro vertical, $v_i\neq0m/s$

**4) Romina	deja	caer	un	desarmador	desde	la	azotea	de	su	casa.	Si	el	desarmador	llega	al	suelo	después	de	2	s.,	calcula:	a)	la	altura	de	la	casa	de	Romina	y	b)	la	velocidad	con	la	que	el	desarmador	llega	al	suelo.**

* Datos:
    * $t=2s$
    * $h=-19.62m$
    * $v_i=0m/s$
    * $v_f=?$
    * $g=9.81m/s^2$

* Fórmulas:
    1. $h=\left(\dfrac{v_i+v_f}{2}\right)t$

    2. $v_f=v_i-g\cdot t$

    3. $h=v_i\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h=v_f\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${v_f}^2={v_i}^2-2\cdot g\cdot h$

* Sustitución

$$h=-\dfrac{1}{2}\cdot (9.81m/s^2)\cdot (2s)^2=-19.62m$$

$$v_f=-(9.81m/s^2)\cdot (2s)=-19.62m/s$$

**5) Raúl	lanza	un	balón	de	basquetbol	verticalmente	hacia	arriba	y	regresa	a	su	punto	de	partida	3.2	s.	
después.	Calcula	a)	la	velocidad	del	lanzamiento,	b)	la	altura	máxima	que	alcanza	la	pelota,	c)	el	
tiempo	de	subida	y	d)	la	velocidad	con	la	que	regresa	a	las	manos	de	Raúl.**

Método 1:
* Datos para el inciso a):
    * $t=3.2s$ (es el tiempo que tarda en subir y bajar; si sólo quisiéramos el tiempo que tarda en subir sería la mitad de $3.2s$)
    * $h=0m$
    * $v_i=?$
    * $g=9.81m/s^2$

Método 2:
* Datos para el inciso a):
    * $t=1.6s$
    * $v_i=?$
    * $v_f=0m/s$ (ésta es cero porque llegó a su altura máxima)
    * $g=9.81m/s^2$

La rapidez con la que regresa el balón a las manos de Raúl es la misma con la que lo aventó pero negativa.

* Datos para el inciso b):
    * $t=1.6s$
    * $h=?$
    * $v_i=15.696m/s$
    * $v_f=0m/s$ (ésta es cero porque llegó a su altura máxima)
    * $g=9.81m/s^2$


* Fórmulas:
    1. $h=\left(\dfrac{v_i+v_f}{2}\right)t$

    2. $v_f=v_i-g\cdot t$

    3. $h=v_i\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h=v_f\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${v_f}^2={v_i}^2-2\cdot g\cdot h$

* Sustitución:

#### Inciso a)

Método 1:

$$h=v_i\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$$

$$h+\dfrac{1}{2}\cdot g\cdot t^2=v_i\cdot t$$

$$v_i=\dfrac{h+\dfrac{1}{2}\cdot g\cdot t^2}{t}$$

$$v_i=\dfrac{0m+\dfrac{1}{2}\cdot (9.81m/s^2)\cdot (3.2s)^2}{(3.2s)}=15.696m/s$$

Método 2:

$$v_i=v_f+g\cdot t=0+(9.81m/s^2)(1.6s)=15.696m/s$$

#### Inciso b)

$$h=\left(\dfrac{15.696m/s+0m/s}{2}\right)(1.6s)=12.55m$$

# Clase 15 

**Promedio:** Valor más probable

**Error absoluto:** Restamos cada medida con el promedio y obtenemos el promedio de éstas.

**Intervalo de confianza:** El promedio $\pm$ el error abosluto.

**Error relativo:** Error absoluto entre el promedio.

In [5]:
2+2

4

In [6]:
rand(5.3:0.1:5.9, 7)

7-element Array{Float64,1}:
 5.6
 5.9
 5.8
 5.3
 5.4
 5.7
 5.8

**Promedio**

In [7]:
5.6+5.9+5.8+5.3+5.4+5.7+5.8

39.5

In [8]:
39.5/7

5.642857142857143

**Nota:** En el promedio tomas el mismo número de decimales que hay en tus medidas originales. Además, redondeas.


* El promedio es 5.6

Los siguientes son la resta "absoluta" de cada medida con el promedio.

0.0, 0.3, 0.2, 0.3, 0.2, 0.1, 0.2

La suma de todos los números anteriores es 1.3

In [9]:
1.3/7

0.18571428571428572

* El error absoluto es 0.18

* El intervalo de confianza es $(5.6\pm0.18)$

Esto quiere decir que lo que estamos midiendo puede medir entre 5.42 y 5.78

In [12]:
0.18/5.6

0.03214285714285715

* El error relativo es $0.032$

* El error relativo porcentual es $3.2\%$

## Tiro parabólico

La velocidad inicial en el tiro parabólico está dado por un vector:

$$ \vec{v_i}=(v_i,\theta) $$

Las componentes horizontal y vertical son

$$ {v_i}_x=v_i\cos\theta $$

$$ {v_i}_y=v_i\sin\theta $$

En el tiro parabólico el movimiento en la horizontal es MRU. Por lo que las fórmulas son

* $ {v_i}_x=\dfrac{d}{t} $
* $ {v_f}_x={v_i}_x $

En el tiro parabólico el movimiento en la vertical es MRUA. Por lo que las fórmulas son

* Fórmulas:
    1. $h=\left(\dfrac{{v_i}_y+{v_f}_y}{2}\right)t$

    2. ${v_f}_y={v_i}_y-g\cdot t$

    3. $h={v_i}_y\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h={v_f}_y\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${{v_f}_y}^2={{v_i}_y}^2-2\cdot g\cdot h$
    
    6. ${v_i}_x=\dfrac{d}{t}$
    
    7. ${v_f}_x={v_i}_x $

**4) Se	lanza	una	pelota	de	golf	con	una	velocidad	(resultante)	de	40	m/s	con	un	ángulo	de	$45^\circ$	con	
respecto	a	la	horizontal.	Determina	a)	el	tiempo	total	de	vuelo	de	la	pelota,	b)	el	tiempo	que	tarda	la	
pelota	en	alcanzar	su	altura	máxima,	c)	la	altura	máxima	que	alcanza	la	pelota	y	d)	su	alcance	horizontal.**


$$ \vec{F}=(40N,45^\circ) $$

* Datos:
    * La velocidad (¡es un vector!) inicial es $\vec{v}_i=(40m/s,45^\circ)$
    * La magnitud de la velocidad inicial es $v_i=40m/s$
    * La dirección de la velocidad inicial es $\theta=45^\circ$

El *tiempo de vuelo* es el tiempo que tarda el proyectil en regresar al piso.

#### Inciso a)

* $t=?$ (tiempo de vuelo)
* $h=0m$
* ${v_i}_y=28.28m/s$ (pero la puedo obtener)
* $g=9.81m/s^2$

$$ {v_i}_y=v_i\sin\theta=(40m/s)\sin45^\circ=28.28m/s $$

$$h={v_i}_y\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$$

$$0=28.28\cdot t-4.905\cdot t^2$$

$$4.905\cdot t^2=28.28\cdot t$$

$$\dfrac{t^2}{t}=\dfrac{28.28}{4.905}$$

$$t=5.76s$$

$$0=t(28.28-4.905\cdot t)$$

* Fórmulas:
    * ${v_i}_y=v_i\sin\theta$
    
    1. $h=\left(\dfrac{{v_i}_y+{v_f}_y}{2}\right)t$

    2. ${v_f}_y={v_i}_y-g\cdot t$

    3. $h={v_i}_y\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h={v_f}_y\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${{v_f}_y}^2={{v_i}_y}^2-2\cdot g\cdot h$
    
    * ${v_i}_x=v_i\cos\theta$
    
    6. ${v_i}_x=\dfrac{d}{t}$
    
    7. ${v_f}_x={v_i}_x $

#### Inciso b)

#### Inciso c)  la altura máxima que alcanza la pelota 

* $t=2.88s$
* $h=40.72 m$ (altura máxima)
* ${v_i}_y=28.28m/s$
* ${v_f}_y=0m/s$
* $g=9.81$

* Fórmulas:
    * ${v_i}_y=v_i\sin\theta$
    
    1. $h=\left(\dfrac{{v_i}_y+{v_f}_y}{2}\right)t$

    2. ${v_f}_y={v_i}_y-g\cdot t$

    3. $h={v_i}_y\cdot t-\dfrac{1}{2}\cdot g\cdot t^2$

    4. $h={v_f}_y\cdot t+\dfrac{1}{2}\cdot g\cdot t^2$

    5. ${{v_f}_y}^2={{v_i}_y}^2-2\cdot g\cdot h$
    
    * ${v_i}_x=v_i\cos\theta$
    
    6. ${v_i}_x=\dfrac{d}{t}$
    
    7. ${v_f}_x={v_i}_x $

#### d)	su	alcance	horizontal.

* $d=166.89m$
* ${v_i}_x=28.28m/s$
* $t=5.76s$

<img src="dibujo.png" align="middle" width="600">

**3) Desde	un	avión	que	vuela	horizontalmente	a	1400	m	de	altura	con	una	velocidad	de	140	m/s	se	deja	
caer	un	bulto	de	alimentos.	Hallar:	a)	el	tiempo	que	tarda	el	bulto	en	llegar	al	suelo,	b)	el	alcance	del	
bulto,	c)	la	magnitud	de	la	velocidad	del	bulto	cuando	choca	con	el	suelo.**

La fuerza es aquel agente físico que provoca que cambie el estado de movimiento de un objeto o su forma.

*Fuerza interna* significa las fuerzas que están sucediendo dentro de un objeto.

## Leyes de Newton

1a	ley: En ausencia de fuerzas, si un objeto se encuentra en reposo, seguirá en reposo; si un objeto se encuentra con una rapidez, seguirá con esta misma rapidez en una trayectoria rectilínea.

2a	ley: Si hay una fuerza , hay un aceleración.
$$ \vec{F}=m\vec{a} $$
	
3a	ley: Si un primer objeto ejerce una fuerza con un segundo objeto, el seguno objeto ejercerá una fuerza de igual magnitud pero sentido contrario.


Se	aplica	una	fuerza	de	550	N	para	mover	un	objeto	de	22	Kg,	¿cuál	será	la	aceleración	producida?	
	


¿Cuál	es	el	peso	de	un	automóvil	cuya	masa	es	de	1.2	toneladas?

1 ton = 1000 kg

$$ w=mg $$